.storybook/gu-preview/GuPreview.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-fallthrough */
+import React, { Fragment, ReactElement } from 'react';
+
+import { styled } from '@storybook/theming';
+import { Icons, IconButton } from '@storybook/components';
+import { knobsData } from '../../src/utils/knobsData';
+
+const IconButtonWithLabel = styled(IconButton)(() => ({
+    display: 'inline-flex',
+    alignItems: 'center',
+}));
+
+const IconButtonLabel = styled.div<{}>(({ theme }) => ({
+    fontSize: theme.typography.size.s2 - 1,
+    marginLeft: 10,
+}));
+
+function constructPreviewUrl() {
+    let baseUrl =
+        'https://www.theguardian.com/sport/2019/jul/28/tour-de-france-key-moments-egan-bernal-yellow-jersey?dcr=false&';
```

Comment:
We actually shouldn't need the `dcr=false` here any more, since forcing is now available on both platforms, no harm though.

Reply:
Good point, I think I'll take it away for now!